convert/src/label_studio_to_badgerdoc/label_studio_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,431 @@
+                self.s3_output_bucket,
+                s3_output_tokens_path,
+            )
+
+            badgerdoc_annotations_path = tmp_dirname / Path(
+                "badgerdoc_annotations.json"
+            )
+            self.badgerdoc_format.export_annotations(
+                badgerdoc_annotations_path
+            )
+            s3_output_annotations_path = self.get_output_annotations_path(
+                importjob_id_created, file_id_in_assets
+            )
+            self.s3_client.upload_file(
+                str(badgerdoc_annotations_path),
+                self.s3_output_bucket,
+                s3_output_annotations_path,
+            )
+
+    def request_jobs_to_create_annotation_job(
+        self,
+        file_id_in_assets,
+        owner,
+        validation_type,
+        deadline,
+        extensive_coverage,
+        annotators,
+        validators,
+    ) -> int:
+        categories = self.request_annotations_to_post_categories()
+        post_annotation_job_url = f"{settings.job_service_url}create_job/"
+        post_annotation_job_body = {
+            "name": f"import_label_studio_job_{uuid4()}",
+            "type": "AnnotationJob",
+            "files": [file_id_in_assets],
+            "datasets": [],
+            "is_draft": False,
+            "validation_type": validation_type,
+            "annotators": annotators,
+            "owners": [owner],
+            "categories": categories,
+            "validators": validators,
+        }
+        if deadline:
+            post_annotation_job_body.update(
+                {"deadline": jsonable_encoder(deadline)}
+            )
+        if extensive_coverage is not None:
+            post_annotation_job_body.update(
+                {"extensive_coverage": extensive_coverage}
+            )
+        LOGGER.debug(
+            "Making a request to create an Annotation Job in 'jobs' to url: %s with request body: %s",
+            post_annotation_job_url,
+            post_annotation_job_body,
+        )
+
+        try:
+            request_to_post_annotation_job = requests.post(
+                url=post_annotation_job_url,
+                headers=self.request_headers,
+                json=post_annotation_job_body,
+            )
+            request_to_post_annotation_job.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            LOGGER.exception(
+                "Failed request to 'jobs' to post AnnotationJob - %s", e
+            )
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail=f"Failed request to 'jobs' to post AnnotationJob",
+            ) from e
+
+        LOGGER.debug(
+            "Got this response from jobs service: %s",
+            request_to_post_annotation_job.json(),
+        )
+        return request_to_post_annotation_job.json()["id"]
+
+    def get_categories_of_links(self, pages_objs):
```

Comment:
Add typings, please

Reply:
I've left List[Any] typehint for pages_objs.
It's not very specific, but it's really difficult to provide specific typehint here.
pages_objs is a list of pydantic_model deeply nested attributes

Reply:
But shouldn't it be a Page object? Like `List[Pages]`, where `Pages` - pydantic class for pages

Reply:
And add a return typing as well, please

Reply:
Actually yes, you're right. Corrected